setup/deploy_cloud_fncs.sh

```diff
@@ -40,8 +40,10 @@ else # Deploy the Cloud Functions
     echo "Deploying Cloud Function: $store_bq_CF_name"
     store_bq_entry_point="run"
 
-    cd .. && cd cloud_functions
-    cd store_BigQuery
+    # the next two lines had the following error: No such file or directory
```

Comment:
> So you'd just need to figure out how those calls need to be updated so that we end up in the right place.

And after looking again, I see that's probably what your changes already do.